.test/meta-commands/out.sh

```diff
@@ -137,6 +137,29 @@ jq -s '
 	)
 ' temp/index.json > temp/index.json.new
 mv temp/index.json.new temp/index.json
+SOURCE_DATE_EPOCH=1709081058 \
+	docker buildx build --progress=plain \
+	--provenance=false \
+	--build-context "base:latest=oci-layout://$PWD/temp/@$(jq -r '.manifests[0].digest' index.json)" \
+	--sbom=generator="$BASHBREW_BUILDKIT_SBOM_GENERATOR" \
+	--output 'type=oci,tar=false,dest=temp_sbom/' \
+	- <<<'FROM base'
+newIndexDigest="$(jq -r '.manifests[0].digest | sub("sha256:";"")' temp_sbom/index.json)"
+sbomManifestDigest="$(jq -r '.manifests[] | select(.platform.os == "unknown").digest | sub("sha256:";"")' temp_sbom/blobs/sha256/$newIndexDigest)"
```

Comment:
Relying on the `unknown` platform makes me nervous -- I'd rather lean on https://docs.docker.com/build/attestations/attestation-storage/ and check for `.annotations["vnd.docker.reference.type"] == "attestation-manifest"` explicitly (ideally we'd _also_ then verify that `vnd.docker.reference.digest` exists and points to a real thing, and maybe even validate some of our assumptions about this clever method, like the layers being identical to our original image, but due to the lack of nice OCI layout tooling I think that's probably too much for now).